File: JANUARY/05-01-2026/1975.py
class Solution:
    def maxMatrixSum(self, matrix: List[List[int]]) -> int:
        total = 0
        neg = 0
        mn = float('inf')
        for row in matrix:
            for x in row:
                if x < 0:
                    neg += 1
                total += abs(x)
                mn = min(mn, abs(x))
        if neg % 2:
            total -= 2 * mn
        return total

File: 2026POTD/JANUARY/05-01-2026/1975.c
long long maxMatrixSum(int** matrix, int matrixSize, int* matrixColSize) {
    long long sum = 0;
    int neg = 0;
    int mn = INT_MAX;
    for (int i = 0; i < matrixSize; i++) {
        for (int j = 0; j < matrixColSize[i]; j++) {
            int x = matrix[i][j];
            if (x < 0) neg++;
            sum += llabs(x);
            if (abs(x) < mn) mn = abs(x);
        }
    }
    if (neg % 2) sum -= 2LL * mn;
    return sum;
}